### PR TITLE
PILOT-1825: Add semantic version tags

### PR DESCRIPTION
**Pull request by mloshakov**
## Summary

Update service to use semantic version tags.
Bump version to 2.0.0.

## JIRA Issues

[PILOT-1825](https://indocconsortium.atlassian.net/browse/PILOT-1825)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -76,6 +78,10 @@ jobs:
           registry: ghcr.io
           username: indoc-devops
           password: ${{ secrets.GHCR_PAT }}
+      - name: Get Version
```
Comment (andricDu) [inline .github/workflows/build-and-publish.yml:81]: extract to its own job

File: .github/workflows/build-and-publish.yml [hunk 2]
```diff
@@ -86,6 +92,7 @@ jobs:
           # generate Docker tags based on the following events/attributes
           tags: |
             type=sha,enable=true,prefix=notification-,suffix=,format=short
+            type=raw,prefix=alembic-,suffix=,value=${{needs.get-version.outputs.app_version}}, enabled=${{needs.extract-branch-name.outputs.branch=='main'}}
```
Comment (andricDu) [inline .github/workflows/build-and-publish.yml:95]: careful with the prefix

Comment (andricDu) [reply .github/workflows/build-and-publish.yml:95]: also `enable` vs `enabled`

**Review by andricDu — CHANGES_REQUESTED** [review]
comments and both images need to be tagged.

**Comment by mloshakov** [comment]
@andricDu Fixed your comments, thanks for review

**Pull request merged**